### PR TITLE
Feat/60 site layout m

### PR DESCRIPTION
**Pull request by Hornebom**
- global site layout for homepage and slug
- hero in two levels

### CODE REVIEW

File: src/components/modular-content/index.njk [hunk 1]
```diff
@@ -13,14 +13,16 @@
...
-  {% for item in items %}
-    {% set itemType = item._modelApiKey %}
-    {% set component = componentByType[itemType] %}
-    {% if component %}
-      {{ component(item, context) }}
-    {% else %}
-      <script>console.warn('Unknown item type "{{ itemType }}"');</script>
-    {% endif %}
-  {% endfor %}
+  <div class="container modular-content__inner">
```
Comment (bob-voorhoede) [inline src/components/modular-content/index.njk:16]: So there will never be any content that spans the full window width?

File: src/components/modular-content/modular-content.scss [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+  position: relative;
+  z-index: 1;
+
+  padding-right: $unit-xsmall;
+  padding-left: $unit-xsmall;
+}
+
+.modular-content__inner {
+  background-color: $color-light;
+  min-height: 100vh;
```
Comment (bob-voorhoede) [inline src/components/modular-content/modular-content.scss]: Is this temporary?

**Review by bob-voorhoede — COMMENTED** [review]
I like it 💚 
Two minor questions, otherwise entirely mergable

File: src/components/modular-content/index.njk (hunk 1 above)
Comment (Hornebom) [reply src/components/modular-content/index.njk:16]: there's no such a block in the design

File: src/components/modular-content/modular-content.scss (hunk 2 above)
Comment (Hornebom) [reply src/components/modular-content/modular-content.scss]: good catch. will remove that

**Pull request merged**